_studio/mfx_lib/encode_hw/hevc/agnostic/g9/hevcehw_g9_weighted_prediction.cpp

```diff
@@ -44,7 +44,7 @@ void WeightPred::SetSupported(ParamSupport& blocks)
 void WeightPred::Query1WithCaps(const FeatureBlocks& /*blocks*/, TPushQ1 Push)
 {
     Push(BLK_CheckAndFix
-        , [this](const mfxVideoParam& /*in*/, mfxVideoParam& par, StorageW& global) -> mfxStatus
+        , [](const mfxVideoParam& /*in*/, mfxVideoParam& par, StorageW& global) -> mfxStatus
```

Comment:
hm. Wasn't that a fix for compiler issue to include 'this'? Not sure which compiler. I think @eistarov worked on it. Can you, please, comment?

Reply:
@dvrogozh, I removed it because clang CI failed due to unused "this" in lambda. Before this code wasn't compiled because of disabled define in mfx_config.h

Reply:
ok. @eistarov likely fixed similar issue in the active code  that's why I remember